src/components/FPagination/FPagination.vue

```diff
@@ -60,79 +52,73 @@ export default {
       default: 10
     }
   },
+
   data: () => ({
     localCurrentPage: 0
   }),
+
   computed: {
-    totalPages() {
+    isFirstPage () {
+      return this.localCurrentPage === 1
+    },
+
+    isLastPage () {
+      return this.localCurrentPage >= this.totalPages
+    },
+
+    totalPages () {
       if (!this.total || !this.perPage) return 0
       return Math.ceil(this.total / this.perPage)
     },
-    items() {
-      return this.currentPage
-    },
-    show() {
-      let result = []
-      let base = this.localCurrentPage
-      let max = this.max
 
-      let factor = Math.ceil(max / 2)
-      let pgFrom = base - factor
-      let pgTo = base <= this.totalPages ? base + factor : this.totalPages
+    pgFrom () {
```

Comment:
Nessa computed você pode fazer o retorno direto.

`return this.localCurrentPage - Math.ceil(this.max / 2)`

Reply:
Feito